viadot/flows/prefect_logs.py

```diff
@@ -69,22 +69,30 @@ def __init__(
 
         self.gen_flow()
 
-    def get_formatted_value_from_timestamp(self, value_type, value) -> str:
+    def get_formatted_value_from_timestamp(
+        self, value_type: str, value: Timestamp
+    ) -> str:
         """
         Function returns cleaned date/time from timestamp in string format (ex. "2022-01-01")
         Args:
-            type (str):  Desired part of the Timestamp value, "date" or "time".
+            value_type (str):  Desired part of the Timestamp value, "date" or "time".
```

Comment:
Unify data types.
line 73 - you have `Timestamp` type. In line 78 is `str` - which is correct?